asistente_ladm_col/gui/supplies/wiz_supplies_etl.py

```diff
@@ -281,6 +285,22 @@ def show_message(self, message, level, duration=10):
         self.bar.clearWidgets()  # Remove previous messages before showing a new one
         self.bar.pushMessage(message, level, duration)
 
+    def show_resumen_etl(self):
```

Comment:
This method does not "show" the summary, right? It "fills" the summary. 
There is a wizard button that shows the summary (going to the next page).